src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background: url("../images/star.svg") no-repeat center;
+  margin: 0 4px 0 0;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background: url("../images/star-active.svg") no-repeat center;
```

Comment:
You can change only background-image

Reply:
Is there any problem to use shorthand property?